packages/bindgen/spec.yml

```diff
@@ -615,6 +620,9 @@ classes:
       simulate_sync_error: '(session: SyncSession&, code: int, message: StringData, type: StringData, is_fatal: bool)'
       consume_thread_safe_reference_to_shared_realm: '(tsr: ThreadSafeReference) -> SharedRealm'
       file_exists: '(path: StringData) -> bool'
+      erase_subscription: '(subs: MutableSyncSubscriptionSet&, sub_to_remove: const SyncSubscription&) -> bool'
+      # This is added due to DescriptorOrdering not being exposed
+      get_results_description: '(results: const Results&) -> StringData'
```

Comment:
This may not be needed any more. Will look into it. Was initially used for getting `Results.description()` without using the base method (`OrderedColleciton.description()`).